src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -82,6 +83,25 @@
     long timerExpire;
     boolean attemptRefreshTokenLocked = false;
 
+    // Threasholds related to when prepared statement handles are cleaned-up. 1 == immediately.
+    /**
+     * The initial default on application start-up for the prepared statement clean-up action threshold (i.e. when sp_unprepare is called). 
+     */    
+    static final public int INITIAL_DEFAULT_SERVER_PREPARED_STATEMENT_DISCARD_THRESHOLD = 10; // Used to set the initial default, can be changed later.
```

Comment:
Sure, is there a specific reason for this other than changing for sake of changing? 

My take was final was analog to const and we commonly have const fields public for clarity.